client/rest-high-level/src/main/java/org/elasticsearch/client/AsyncSearchClient.java

```diff
@@ -63,4 +66,55 @@ public Cancellable submitAsyncSearchAsync(SubmitAsyncSearchRequest request, Requ
                 AsyncSearchResponse::fromXContent, listener, emptySet());
     }
 
+    /**
+     * Get an async search request.
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html"> the docs</a> for more.
+     *
+     */
+    public AsyncSearchResponse getAsyncSearch(GetAsyncSearchRequest request, RequestOptions options) throws IOException {
```

Comment:
I wonder if we can rename into `get` and `getAsync` since we're already inside the `AsyncSearchClient` ?

Reply:
Sounds good, no problem with that...